src/less/listbox-button/stories/subtitle.stories.js

```diff
@@ -0,0 +1,441 @@
+export default { title: "Skin/Listbox Button/With Subtitle" };
+
+export const collapsedUnselected = () => `
+<span class="listbox-button">
+    <button class="btn btn--form" aria-expanded="false" aria-haspopup="listbox">
+        <span class="btn__cell">
+            <span class="btn__label">Color: </span>
+            <span class="btn__text">-</span>
+            <svg class="icon icon--chevron-down-12" focusable="false" height="8" width="8" aria-hidden="true">
+                <use href="#icon-chevron-down-12"></use>
+            </svg>
+        </span>
+    </button>
+    <div class="listbox-button__listbox">
+        <div class="listbox-button__options" role="listbox">
+            <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">Red</span>
```

Comment:
In terms of styling, I think it may make more sense to wrap the `value` and `subtitle` in a span or div which flexes vertically. This will probably take away some grid complexity while allowing for icons before _and_ after by default. Let me know what you think of something like this:
```marko
div.listbox-button__option
  if=prefixIcon
    ${prefixIcon}

  if=subtitle
    div.listbox-button__value
      span -- ${value}
      span -- ${subtitle}
  else
    span.listbox-button__value -- ${value}

  if=postfixIcon
    ${postfixIcon}
```

Reply:
In ebayui this won't be an issue
I think the tricky part is the makeup 
Because in makeup we get the text of the value element. Currently we are getting the text of the option but we will have to change it to only get the value. If subtitle is nested there is no way to get the inner text of just the value. (Or you can but its a bit more complex)
We could probably add an attribute to support that if needed maybe? Like on option we can do data-option-text. 

One other issue potentially is we dont allow word wrap in listbox value but we do allow it for subtitle as well as subtitle is on a new line. 
Unfortunately i think this is the best way to do it from that perspective.

Reply:
Ok, so what about
```marko
div.listbox-button__option
  if=prefixIcon
    ${prefixIcon}

  if=subtitle
    div.listbox-button__subvalue
      span.listbox-button__value -- ${value}
      span.clipped -- . 
      span -- ${subtitle}
  else
    span.listbox-button__value -- ${value}

  if=postfixIcon
    ${postfixIcon}
```

Reply:
The thing i don't like about your suggestion is that we have to support two layouts rather than just one. 
I'm also not sure if itll work since we need to support not wrapping of the title (and ellipsis) but wrapping of the subtitle.

Reply:
Also, we are introducing flags in phone-input where we might need to list `svg`. We need to think through this and apply changes both on eBay UI and makeup level.